hikari/_about.py

```diff
@@ -36,5 +36,5 @@
 __issue_tracker__ = "https://github.com/hikari-py/hikari/issues"
 __license__ = "MIT"
 __url__ = "https://github.com/hikari-py/hikari"
-__version__ = "2.0.0"
+__version__ = "0.0.0"
```

Comment:
what's with this ver change?

Reply:
I forgot to open a pr that fixes this. I was playing around with a version that we dont need to change and can stay constant between all versions. This would also help us separate between official versions and non-official ones.

Reply:
Opened #426 for this